Getting started tutorial: Create a `package.json` to install the npm packages locally (#544)
scala-js/scala-js-website: doc/tutorial/basic/index.md

jsEnv := new org.scalajs.jsenv.jsdomnodejs.JSDOMNodeJSEnv()
{% endhighlight %}

> This will use the [`jsdom`](https://github.com/jsdom/jsdom) library to simulate a DOM in Node.js. You need to install it separately.

Reviewer: Try to preserve one line = one sentence:
```suggestion
This will use the [`jsdom`](https://github.com/jsdom/jsdom) library to simulate a DOM in Node.js.
You need to install it separately.
```

> Best is to install this locally for this project. Therefore, initialise a `package.json` using the default values:

Reviewer: ```suggestion
It is usually best to install it locally for this project.
You can initialize a private `package.json` as follows:
```

>     $ npm init -y

Reviewer: I believe `npm init private` would be more appropriate in this case.

Author: I am not familiar with that.  As long as it does not ask questions during a tutorial.
